eng/Build.props

```
@@ -173,6 +173,7 @@
                           $(RepoRoot)src\Shared\**\*.*proj;
                           $(RepoRoot)src\Tools\**\*.*proj;
                           $(RepoRoot)src\Logging.AzureAppServices\**\src\*.csproj;
+                          $(RepoRoot)src\Logging.W3C\**\src\*.csproj;
```

The sample and test projects shouldn't build if you do this. If they are built (and the tests run), I'd like to understand how&hellip;
```suggestion
                          $(RepoRoot)src\Logging.W3C\**\*.csproj;
```

---

https://github.com/dotnet/aspnetcore/pull/33251/commits/fcf767132a91adb16682c7b8afc1f4a628194c23